zanshinsdk/client.py

```diff
@@ -199,92 +207,344 @@ def _request(self, method: str, path: str, params=None, body=None) -> httpx.Resp
-                                      severities: Optional[Iterable[AlertSeverity]] = None, page: int = 1,
-                                      page_size: int = 100) -> Dict:
+    def update_organization(self, organization_id: Union[UUID, str], name: Optional[str], picture: Optional[str], email: Optional[str]) -> Dict:
         """
-        Internal method to retrieve a single page of alerts from an organization.
+        Update organization given its ID.
+        <https://api.zanshin.tenchisecurity.com/#operation/editOrganizationById>
         :param organization_id: the ID of the organization
-        :param scan_target_ids: optional list of scan target IDs to list alerts from, defaults to all
-        :param states: optional list of states to filter returned alerts, defaults to all
-        :param severities: optional list of severities to filter returned alerts, defaults to all
-        :param page: page number, starting from 1
-        :param page_size: page size in number of alerts
-        :return:
+        :param name: the Name of the oranization
+        :param picture: the picture of the organization
+        :param email: the e-mail contact of the organization
+        :return: a dict representing the organization object
         """
-        validate_int(page, min_value=1, required=True)
-        validate_int(page_size, min_value=1, required=True)
         body = {
-            "page": page,
-            "pageSize": page_size
+            "name": name,
+            "picture": picture,
+            "email": email
         }
-        if organization_id:
-            body['organizationId'] = validate_uuid(organization_id)
-        if scan_target_ids:
-            if isinstance(scan_target_ids, str):
-                scan_target_ids = [scan_target_ids]
-            else:
-                validate_class(scan_target_ids, Iterable)
-            body['scanTargetIds'] = [validate_uuid(x) for x in scan_target_ids]
-        if states:
-            validate_class(states, Iterable)
-            body['states'] = [validate_class(x, AlertState).value for x in states]
-        if severities:
-            validate_class(severities, Iterable)
-            body['severities'] = [validate_class(x, AlertSeverity).value for x in severities]
-        return self._request("POST", "/alerts", body=body).json()
+        return self._request("PUT", f"/organizations/{validate_uuid(organization_id)}", body=body).json()
 
-    def iter_organization_alerts(self, organization_id: Union[UUID, str],
-                                 scan_target_ids: Optional[Iterable[Union[UUID, str]]] = None,
-                                 states: Optional[Iterable[AlertState]] = None,
-                                 severities: Optional[Iterable[AlertSeverity]] = None, page_size: int = 100) -> \
-            Iterator[Dict]:
+    def iter_members(self, organization_id: Union[UUID, str]) -> Iterator[Dict]:
```

Comment:
I think `iter_organization_members` would be clearer and more in line with the other methods such as `iter_organization_alerts`.

Reply:
Same for all of the other member methods.